distributed/tests/test_stress.py

```diff
@@ -166,3 +166,25 @@ def test_stress_communication(c, s, *workers):
 
     result = yield future._result()
     assert isinstance(result, float)
+
+
+@slow
+@gen_cluster(client=True, ncores=[('127.0.0.1', 1)] * 8, timeout=10000)
+def test_communication_heavy_graph(c, s, *workers):
+    da = pytest.importorskip('dask.array')
```

Comment:
By the way, it seems distributed already imports Numpy and other stuff unconditionally:
```
>>> import distributed
>>> import sys
>>> "numpy" in sys.modules
True
>>> "pandas" in sys.modules
True
```